extensions/ftp/src/main/java/org/mule/extension/ftp/internal/sftp/connection/SftpConnectionProvider.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.ftp.internal.sftp.connection;
+
+import org.mule.extension.ftp.api.FtpConnector;
+import org.mule.extension.ftp.api.sftp.SftpFileSystem;
+import org.mule.extension.ftp.internal.AbstractFtpConnectionProvider;
+import org.mule.runtime.api.connection.ConnectionException;
+import org.mule.runtime.api.connection.ConnectionHandlingStrategy;
+import org.mule.runtime.api.connection.ConnectionHandlingStrategyFactory;
+import org.mule.runtime.api.connection.PoolingListener;
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.Parameter;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+import org.mule.runtime.extension.api.annotation.param.display.Password;
+
+/**
+ * An {@link AbstractFtpConnectionProvider} which provides instances of
+ * {@link SftpFileSystem} from instances of {@link FtpConnector}
+ *
+ * @since 4.0
+ */
+@Alias("sftp")
```

Comment:
this alias should not match the constant you mention. If now happens to match, but they shouldn't be tied together